starter_code/views/beers.hbs

```diff
@@ -0,0 +1,8 @@
+<div class="theBeerList">
+{{#each beers}}
+    {{> beers this}}
```

Comment:
Instead of beers, it should be the name of the partial file. In your case is beerPartial